src/main/kotlin/no/nav/syfo/personstatus/api/v2/model/PersonOversiktStatusDTO.kt

```diff
@@ -24,6 +24,7 @@ data class PersonOversiktStatusDTO(
     val arbeidsuforhetvurdering: ArbeidsuforhetvurderingDTO?,
     val friskmeldingTilArbeidsformidlingFom: LocalDate?,
     val oppfolgingsoppgave: OppfolgingsoppgaveDTO?,
+    val senOppfolgingKandidatUbehandlet: Boolean,
```

Comment:
Hva med å ta inspirasjon fra de over og sende med en DTO? Selv om vi ikke vet om skjemasvar kan det være andre greier vi er interesserte i som vi kanskje vil vise i oversikten? I første omgang så kan vi jo for eks. ta med status på kandidaten 😊
```suggestion
    val senOppfolgingKandidat: SenOppfolgingDTO?,
```

```typescript
data class SenOppfolgingDTO {
    status: Status
}
```

Reply:
Hmm eneste problemet er vel at vi ikke vet hva statusen er? Eller vi "vet" jo at når den er `true` i databasen så betyr det `status: KANDIDAT`. Men tanken min var å ikke spørre `ismeroppfølging` igjen om statusen her nå i første omgang 🤔 Vi kan evt utvide med det litt senere?

Reply:
Vi vet hvis vi spør `ismeroppfolging`. Kan godt utvide med senere 😊 Regner med at når vi etterhvert får oversikt over de som har svart og ikke så ønsker vi å vise dette i oversikten.